starport/pkg/dirchange/dirchange.go

```diff
@@ -115,9 +119,15 @@ func checksumFromPaths(workdir string, paths []string) ([]byte, error) {
 	}
 
 	if noFile {
-		return []byte{}, errors.New("no file in specified paths")
+		return []byte{}, &ErrNoFile{}
 	}
 
 	// compute checksum
 	return hash.Sum(nil), nil
 }
+
+type ErrNoFile struct{}
```

Comment:
Can we use an error variable instead to keep it simple? Struct errors are useful when there are some additional data to carry around about the error.